go.mod

```diff
@@ -5,6 +5,7 @@ go 1.18
 require (
 	github.com/dustin/go-humanize v1.0.0
 	github.com/flynn/go-docopt v0.0.0-20140912013429-f6dd2ebbb31e
+	github.com/google/gofuzz v1.2.0
```

Comment:
what brought this in? 🤔

Reply:
Used here - https://github.com/theupdateframework/go-tuf/pull/275/files#diff-51e5e0f62cd2e01cdd3dfa2f7c2a67cb9e0ed0b18db6fe4a23afd2e33264ef42R60 - But due to the fact that you enabled go1.18 since I started this PR, I could switch to native fuzz tests.

Reply:
I missed that, thanks! I'd be happy to approve (and merge) this as-is, we can port to native fuzz tests as a follow-on item.

Reply:
Filed as https://github.com/theupdateframework/go-tuf/issues/346